databases/week2/homework_part4.sql

```diff
@@ -0,0 +1,75 @@
+-- Part 4: Creating a database
```

Comment:
really good work, you seem to have grasped the database notion very well.

Reply:
Thanks a lot. I'm getting better the database notion then two weeks ago, when we have started)). But still so many questions. Can I ask you a one question.You say, that on the diagrams, Customer-Address is one to many and not many to one. Ok. I'm agree that one customer of course can have more than only one address. But an address can also belongs to more than one customer. For exemple, all family members can be a customers of a company and have the same address. Am I wrong? Is it gonna cause a problem?

Reply:
That is a really good point, that really depends on how the database is going to record this address, I believe that it normally treat this as different addresses. But is an edge case.